native/src/hooks/useUserLocation.ts

```diff
@@ -115,11 +110,8 @@ const useUserLocation = (useSettingsListener = false): LocationInformationType =
     return () => undefined
   })
 
-  const isReadyOrLoading = locationState.status === 'ready' || locationState.message === 'loading'
```

Comment:
This one was very sure responsible to avoid the location button in the corner to behave strangely. When you remove this and try to request the location it will always go back into the state of noPermission and then blink between searching and available. I think this should either be readded or you propose a better solution :)

Reply:
my gps position does not work for integreat app on real android device :(
Maybe someone else could test this. On Emulator i couldn't see an issue though after refactoring

Reply:
@sarahsporck please check again, I changed the implementation a little bit and tested this on native. For me everything works fine now.